Feature/dump command (#114)
cf-control/cf-control: README.md

                     the configuration of your cf instance.
```

##### dump [SUBCOMMAND]

```

>   all               Resolve $ref-occurrences in an entire configuration file and print the result.

Reviewer: IMO a complete `dump` would've been sufficient.

Reviewer: ```suggestion
  all               Read complete configuration, resolve all $refs and print the result to the console.
```

I think this entire description is misleading, though. What's an "entire" or "complete" configuration? Why are we still introducing new code about the soon-to-be-removed services/applications/...?

I really think it just creates complexity, and is likely going to be removed soon enough again.

Author: Yes, by only supporting a single dump command, we could easily remove complexity. I will ask the POs in today's meeting about that.

Reviewer 2: It would be good, to see the complexity/size of a yaml file from real user.
If the size and complexity of a yaml may be really big, I think for some users those subcommands would be great.

Author: The POs decided to leave them in for now. In a later sprint we might remove the subcommands again.

Author: After talking to the industry partner, the POs now decided that we should no longer support the sum-commands.

>   services          Resolve $ref-occurrences in a service configuration file and print the result.

Reviewer: ```suggestion
  services          Read space developers configuration, resolve all $refs and print the result to the console.
```

>   space-developers  Resolve $ref-occurrences in a space-developers configuration file and print the result.

Reviewer: ```suggestion
  space-developers  Read space developers configuration, resolve all $refs and print the result to the console.
```

>   applications      Resolve $ref-occurrences in an application configuration file and print the result.

Reviewer: ```suggestion
  applications      Read applications configuration, resolve all $refs and print the result to the console.
```
